test/test.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-env node */
+/* eslint-disable no-console */
+var count, config,
```

Comment:
Hm.. I think most of these are meant to be local to the forEach callback, right? E.g. `config` isn't used in the initialisation scope.